src/Tensor.jl

```diff
@@ -151,6 +151,9 @@ function Base.view(t::Tensor, inds::Pair{Symbol,<:Any}...)
     end
 end
 
+Base.adjoint(t::Tensor{T,1,A}) where {T,A<:AbstractVector{T}} = Tensor(conj(parent(t)), labels(t))
+Base.adjoint(t::Tensor{T,2,A}) where {T,A<:AbstractMatrix{T}} = Tensor(conj(parent(t)), reverse(labels(t)))
```

Comment:
If it is just applying the conjugate, should it reverse the labels?

Reply:
Yes, maybe it's better not to `reverse` the `labels`. I will update that.

So do you agree it's a good idea to put `conj` instead of `adjoint` here?

Reply:
Yes, that's true. We can take a note (or add a comment) and consider this in the future.